converter/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -95,7 +95,12 @@
 	QUALITY_MEASURE_ID_INCORRECT_PERFORMANCE_UUID(60, "The eCQM (electronic measure id: %s) has a %s with an "
 			+ "incorrect UUID of %s", true),
 	QUALITY_MEASURE_ID_MISSING_SINGLE_PERFORMANCE_RATE(61, "A Performance Rate must contain a single "
-			+ "Performance Rate UUID");
+			+ "Performance Rate UUID"),
+	CPC_PLUS_TOO_FEW_QUALITY_MEASURE_CATEGORY(62,
```

Comment:
There will be error id collisions between the new errors added in this PR and the errors added in https://github.com/CMSgov/qpp-conversion-tool/pull/445.  Please work with @halprin to ensure that both PRs have errors with unique ids.